notifications.go

```diff
@@ -148,6 +148,27 @@ type NotificationAvailableAlertsResponse struct {
 	Result NotificationsGroupedByProduct
 }
 
+// NotificationHistory describes the history
+// of notifications sent for an account.
+type NotificationHistory struct {
+	Id            string    `json:"id"`
+	Name          string    `json:"name"`
+	Description   string    `json:"description"`
+	AlertBody     string    `json:"alert_body"`
+	AlertType     string    `json:"alert_type"`
+	Mechanism     string    `json:"mechanism"`
+	MechanismType string    `json:"mechanism_type"`
+	Sent          time.Time `json:"sent"`
+}
+
+// NotificationHistoryResponse describes the notification history
+// response for an account for a specific time period.
+type NotificationHistoryResponse struct {
+	Response
+	ResultInfo
```

Comment:
it doesn't look like we're using `ResultInfo` anywhere; did you intend to?

Reply:
My understanding is if the API returns pagination, it is marshalled to this as well. Please correct me if I am wrong.

Reply:
yep, that's all good. my question was more pointed at the fact the previous return type was only ever returning the `Result` and `error` 🙂 what do you think of having this method return 3 parameters and the result info is in it's own parameter as opposed to putting it all together? The SDK is a little bit of a mishmash of approaches but this does allow people to disregard the result info if they don't want to explicitly paginate.

Reply:
I see what you are saying. I have now changed it to return three parameters separately and also have included an pagination argument to paginate if the users want to.